README.md

```diff
@@ -1,4 +1,125 @@
-# Python boilerplate for GitHub tasks
+# Shop trip
 
 - Read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md) before start
-- Implement the task described [here](app/main.py)
+
+You want to create an application that helps customers to choose the cheapest
+trip for the products.
+
+There is `config.json` file that contains:
+- `FUEL_PRICE` - Price for 1 liter of fuel in dollars.
+- `customers` - list of dictionaries with information about each customer.
+- `shops` - list of dictionaries with information about each shop in the city. 
+
+You have information about 
+
+Customers:
+- name
+- products he wants to buy
+- location
+- money
+- car
+  - brand
+  - volume of fuel consumption for 100 kilometers.
+
+Shops:
+- name
+- location
+- products that shop provides
+
+Write `shop_trip` function that doesn't take any argument,
+where customers calculate how 
+much will cost trip for the products in every shop and pick
+the cheapest one and ride there if they have enough money.
```

Comment:
Could you add it, please?
`The cost of the trip consists of two parts: the cost of all products to buy plus the cost of fuel to get to the shop.`